core/service/src/engine/threshold/service/kms_impl.rs

```diff
@@ -537,8 +548,8 @@ where
         let epoch_id_prss: EpochId = RequestId::try_from(PRSS_INIT_REQ_ID.to_string())?.into(); // the init epoch ID is currently fixed to PRSS_INIT_REQ_ID
         let default_context_id = *DEFAULT_MPC_CONTEXT;
         tracing::info!(
-            "Initializing threshold KMS server and generating a new PRSS Setup for {}",
-            threshold_config.my_id
+            "Initializing threshold KMS server and generating a new PRSS Setup for private storage prefix {:?}",
```

Comment:
This is not just the prefix, right?
```suggestion
            "Initializing threshold KMS server and generating a new PRSS Setup for private storage {:?}",
```

Reply:
fixed in 174481a1a9d0b64e62a7d15faf0937b48a153211